database/frontend/react.json

```
@@ -7,6 +7,14 @@
     "subcategory": "react",
     "language": "english"
   },
+  {
+    "name": "React js",
+    "description": "Free react course - Learn fundamentals",

```

[Nitpick] This is a great start, but devs might want to know why they should take this course. Consider adding that to your description. 
```suggestion
    "description": "Free react course - Learn fundamentals",
```

---

@CBID2 want to say to improve description?
Because the suggested changes are the same.

---

I gave you ideas on _how_ to make changes @Parmesh119. You just have to do them.